### PR TITLE
Translate amp story best practices into Korean

### PR DESCRIPTION
**Pull request by techhtml**
I'd translated amp-story best practices into Korean.
PTAL :)

### CODE REVIEW

**Comment by techhtml** [comment]
@captain-lee Could you review this translation?

File: content/docs/design/amp_story_best_practices@ko.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+     <td>권장하는 최적화 방식:
+      <ul>
+        <li>MP4에서는 <code>-crf 23</code>를 사용합니다.</li>
+        <li>WEBM에서는 <code>-b:v 1M</code>을 사용합니다.</li>
+      </ul>
+     </td>
+    </tr>
+    <tr>
+     <td><a href="https://github.com/google/shaka-packager">Shaka Packager</a></td>
+     <td>재생 목록을 포함한 HLS 포맷을 출력 가능한 엔코더입니다.
```
Comment (captain-lee) [inline content/docs/design/amp_story_best_practices@ko.md]: 원문에 다른 포맷도 출력가능하다는 뉘앙스가 있어서 ("that can also output"), '포맷을' -> '포맷도'로 살짝 바꾸면 어떨까요?

**Comment by techhtml** [comment]
@captain-lee 수정하였습니다!

I fix some sentences with review.

**Comment by pbakaus** [comment]
thanks for the contribution!

**Pull request merged**